docs/concepts/graph/edge.md

```diff
@@ -16,3 +16,45 @@ All of the resources in AWS are placed in a region. The region is one node in th
 If we want to find all resources in the graph, we need to walk _outbound_ (following the edges in direction of the arrow).
 
 If we want to know the account of a specific resource, we need to walk _inbound_ (following the edge in reverse direction of the arrow) in the graph until we find an account.
+
+## Edge Traversal Terminology
+
+In Resoto and this documentation you will find four terms that we use to describe traversals on inbound and outbound edges from and to nodes. You can learn more about the [traversal commands here](../../reference/search/traversals#commands).
+
+In the following examples we always use _one node_ as the _current node_. This is depending on the selection or search, it could also be _multiple nodes_.
```

Comment:
Can you clarify what this means? It's a bit confusing.

Reply:
> Can you clarify what this means? It's a bit confusing.

What part do you mean? The second paragraph or the first paragraph?

If you have a suggestion how to make it more clear it would be great! Otherwise it could eventually also just be removed.

Reply:
I was referring to the highlighted line about one/current node.